### DISCUSSION TITLE
Failed to download resource "freetds"

### DISCUSSION THREAD

**Discussion by shoorace** [shivammathur/homebrew-php#2105] [Q&A / Help]
Hi,

When I failed to install PHP 8.0, 8.1, 8.2, etc and I got the following error:
```
Error: php@8.0: Failed to download resource "freetds"
Download failed: Homebrew-installed `curl` is not installed for: https://www.freetds.org/files/stable/freetds-1.4.6.tar.bz2
```

Please find my infrastructure details:
Hardware: M1 chip
OS: MacOS Big Sur v11.4
Homebrew: 4.1.19
xcode-select version 2384.

Please suggest. Thanks!

Regards,
Suresh

**Comment by shivammathur** [comment]
macOS Big Sur is no longer supported by Homebrew. Please upgrade your OS.
